gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageImplTest.java

```diff
@@ -555,6 +564,7 @@ public void create_gcsItemInfo_metadataEquals() throws IOException {
                         /* finishWrite= */ true))
                 .build()
                 .toArray());
+    trackingGcs.delegate.close();
```

Comment:
this wont close if there is a test failure, correct?

Reply:
yeah, but I am not too much worried. Test failure will any how will take priority over resource cleanup.